triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/ServiceTest.scala

```diff
@@ -181,6 +189,30 @@ class ServiceTest extends AsyncFlatSpec with Eventually with Matchers with Postg
     }
   }
 
+  def parseTriggerStatus(resp: HttpResponse): Future[Vector[String]] = {
+    for {
+      JsObject(fields) <- parseResult(resp)
+      Some(JsArray(list)) = fields.get("logs")
+      statusMsgs = list map {
+        case JsArray(Vector(JsString(_), JsString(msg))) => msg
+        case _ => fail("""Unexpected format in the "logs" field""")
+      }
+    } yield statusMsgs.reverse // Order with least recent at front.
```

Comment:
It seems to make more sense to append to the logs in the first place and avoid reversing here.